x-pack/plugins/canvas/public/services/legacy/index.ts

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
```

Comment:
Note to reviewers: this was moved from `public/services` but Github didn't understand that. 🤷🏻‍♂️